packages/preview/o-rly-typst/0.1.0/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "o-rly-typst"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["Dei Layborer"]
+license = "GPL-3.0"
+compiler = "0.11.1"
+description = "A package for creating O'Rly- / O'Reilly-type cover pages"
+repository = "https://github.com/dei-layborer/o-rly-typst"
+categories = ["components", "layout", "fun"]
+exclude = ["$$$-gimmie.svg", "made-with-(2s)-2,6-diamino-n-[(2s)-1-phenylpropan-2-yl]hexanamide-n-[(2s)-1-phenyl-2-propanyl]-l-lysinamide.svg", "example.png"]
```

Comment:
I think the second file name here have changed in the repository.

Reply:
Thanks for catching this, you're right ... the build process seemed to take issue with the filename.